src/components/learner-credit-management/BudgetStatusSubtitle.jsx

```diff
@@ -8,19 +9,15 @@ import { GroupAdd, Groups, ManageAccounts } from '@openedx/paragon/icons';
 
 import { useIntl } from '@edx/frontend-platform/i18n';
 import {
-  formatDate,
-  useEnterpriseCustomer,
-  useEnterpriseGroup,
-  isLmsBudget,
+  formatDate, useEnterpriseCustomer, useEnterpriseGroup, isLmsBudget,
 } from './data';
 
 const BudgetStatusSubtitle = ({
   badgeVariant, status, isAssignable, term, date, policy, enterpriseUUID, isRetired,
 }) => {
   const { data: enterpriseGroup } = useEnterpriseGroup(policy);
+  const customGroup = !isEmpty(policy?.groupAssociations) && !enterpriseGroup?.appliesToAllContexts;
```

Comment:
nit: I think you could say `!isEmpty(enterpriseGroup) && !enterpriseGroup?.appliesToAllContexts` here instead if you wanted to